app/controllers/api/v2/ext_app/deployments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V2::ExtApp::DeploymentsController < Api::V2::ExtApp::BaseController
+  def bulk_apply
```

Comment:
who will call this API and for what purpose?

Reply:
cc @fadlinurhasan.

Reply:
BaritoMarket will call this API to create deployments for Infrastructure in Pathfinder mono (With this API, pf-mono will generate containers based on deployment value). Previously, when we create an app group on BaritoMarket all components created inside Infrastructure. These Deployments replace InfrastructureComponents for handle containers.

Reply:
oh then why `bulk_apply` not `create` to follow RESTful semantic?

Reply:
We had a discussion with the team that we will have one API for creating and updating deployments. From the Barito side, we generate manifests(list of deployments) as a parameter for this API. 
Do you have any additional statements for this mas @ekarisky ?

Reply:
For pathfinder, its should have at least 2 APIs exposed, `create` for single deployment and `batch` for multiple deployment alongside with respective responses. This is good read about it https://www.codementor.io/blog/batch-endpoints-6olbjay1hd and https://stackoverflow.com/questions/28596688/rest-api-bulk-create-or-update-in-single-request

Reply:
sure make sense
based on the doc that Eka shared, then just use `batch` as the name?